source/clear-linux/get-started/compatibility-check.rst

```diff
@@ -3,26 +3,25 @@
 Check processor and EFI firmware compatibility with Clear Linux OS for Intel® Architecture
 ======================================================================
 
-On a system that is currently running a Linux operating system, follow the instructions below to determine if your target system is capable of running |CL|\*.  Otherwise, see the Note section below for other options.  
+On a system that is currently running a Linux operating system, follow the instructions below to determine if your target system is capable of running |CL|.  Otherwise, run |CL| as a `live image`_ and perform the steps below.  
```

Comment:
Line length 78chars. Use a reference label to connect to a file within the same repo. If you want to use the referenced file's title, you do as the first line below. To use your own text, use the second line below.
Otherwise, :ref:`live-image` and perform the steps below.
Otherwise, :ref:`run Clear Linux as a live image <live-image>` and perform the steps below.